### PR TITLE
Add new sections

### PR DESCRIPTION
**Pull request by montsecruz9**
In this pull request, I did the following: 

- [ ] Update index.html
- [ ] Update style.css
- [ ] Add images to the assets folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
...
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
+          </div>
+        </article>
```
Comment (Cabraham1) [inline index.html:192]: - [ ] Nice work Implementing the portfolio cards and layout, upon careful examination I notice your design is not correspond with the Figma design, some of the key things you need to implement
 - [ ] you content (margin)
 - [ ] padding 

- [ ] Your Design
image

- [ ] Figma Design
image

File: index.html [hunk 2]
```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
...
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
```
Comment (Cabraham1) [inline index.html:97]: - [ ] "Nice work on the code! It appears to be a button with the text 'See project' inside it. Here are a few suggestions for improvement:

- [ ] Consider adding an href attribute to the <a> tag within the button. The href attribute specifies the URL that the button should link to. For example, <a href="project.html">See project</a>.

File: index.html [hunk 3]
```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
...
```
Comment (Cabraham1) [inline index.html:128]: - [ ] "Nice work on the code! It appears to be a button with the text 'See project' inside it. Here are a few suggestions for improvement:

- [ ] Consider adding an href attribute to the <a> tag within the button. The href attribute specifies the URL that the button should link to. For example, <a href="project.html">See project</a>.

File: index.html [hunk 4]
```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
...
```
Comment (Cabraham1) [inline index.html:159]: - [ ] "Nice work on the code! It appears to be a button with the text 'See project' inside it. Here are a few suggestions for improvement:

- [ ] Consider adding an href attribute to the <a> tag within the button. The href attribute specifies the URL that the button should link to. For example, <a href="project.html">See project</a>.

File: index.html [hunk 5]
```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
...
```
Comment (Cabraham1) [inline index.html:190]: - [ ] "Nice work on the code! It appears to be a button with the text 'See project' inside it. Here are a few suggestions for improvement:

- [ ] Consider adding an href attribute to the <a> tag within the button. The href attribute specifies the URL that the button should link to. For example, <a href="project.html">See project</a>.

File: index.html [hunk 6]
```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
...
+						>
+							<h3 class="about_page__subtitle">Skills</h3>
+							<img
+								src="assets/arr_r.svg"
+								alt="Arrow right icon"
+							/>
+						</header>
+					</div>
+					<hr />
+				</div>
```
Comment (Cabraham1) [inline index.html:275]: - [ ] Nice work Implementing the portfolio cards and layout, upon careful examination I notice your design does not correspond with the Figma design, some of the key things you need to implement
 - [ ] flex
 - [ ] margin
 - [ ] padding 

- [ ] Your Design
image


- [ ] Figma Design
image

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @montsecruz9   :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I notice your Pull request has a merge conflict, i kindly suggest you resolve the conflicts soon 
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.


# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (montsecruz9) [reply index.html]: Hi @Cabraham1 ! Thanks for your comments :) Do you have any suggestions on how to put those arrows the way they are like in the figma design?

File: index.html [hunk 7]
```diff
@@ -1,69 +1,269 @@
...
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+              <div>
+                <a class="button_primary" href="#">See project</a>
+              </div>
```
Comment (Shedrack-Sunday) [inline index.html:96]: - [ ] Nice efforts here 👏 . But please remove the underline that you have in the button. The idea is to have a close replica of the design. 👍 

## Figma:
image

## Your design:
image

File: index.html [hunk 8]
```diff
@@ -1,69 +1,269 @@
...
+              <p class="portfolio__description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
```
Comment (Shedrack-Sunday) [inline index.html:186]: - [ ] You made progressive efforts which I must commend 👏 . However, I suggest you make a few adjustments to this section of the cards in reference to the Figma design. 
### Points to note:
- The `h4` font is `Poppins`. kindly adjust it to fit the design given
- Kindly align the `ul` section that has languages to move to the left and align with the card elements

- [ ] Please consider revising your `flexbox` materials for the alignment 👍 __

### Figma:
image

### Your work:
image

File: index.html [hunk 9]
```diff
@@ -1,69 +1,269 @@
...
+            </ul>
+            </div>
+            <div>
+              <a  class="button_primary">Get my resume</a>
+            </div>
+            <div class="category-about_page">
+              <header
+                class="about_page_header flex-row justify-between align-center"
+              >
+                <h3 class="about_page_subtitle">Languages</h3>
```
Comment (Shedrack-Sunday) [inline index.html:220]: - [Optional] Kindly consider adjusting the alignment of some of the features of this section which I've highlighted in the image below to have a replica of what's provided on the Figma design. You'll agree with me it does not give a better user experience 👍 

### Your work:
image

### Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @montsecruz9     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudus on fixing some changes suggested by the last reviewer  ✔️ 
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 8 above)
Comment (AdnanOlarmmi) [reply index.html:186]: > * [ ]  You made progressive efforts which I must commend 👏 . However, I suggest you make a few adjustments to this section of the cards in reference to the Figma design.
> 
> ### Points to note:
> * The `h4` font is `Poppins`. kindly adjust it to fit the design given
> * Kindly align the `ul` section that has languages to move to the left and align with the card elements
> * [ ]  Please consider revising your `flexbox` materials for the alignment 👍 __
> 
> ### Figma:
> image
> 
> ### Your work:
> image

- [ ] For the above comment, kindly fix the space after `CANOPY`, it seems the spaces between the elements are equal kindly adjust it.

Here is your current design.

image

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @montsecruz9, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Fixed most reviewer's changes.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -1,69 +1,269 @@
...
+                </li>
+                <li class="about_page__item">
+                  <img
+                    src="assets/Ellipse3.png"
+                    alt="css image"
+                  />
+                  CSS
+                </li>
+              </ul>
+            </div>
```
Comment (Tufahel) [inline index.html:246]: - [ ] ***[OPTIONAL]*** `Though it is optional it is a very important bug that you need to fix.` 
- Kindly consider adjusting the alignment of some of the features of this section which I've highlighted in the image below to have a replica of what's provided on the Figma design. You'll agree with me it does not give a better user experience 👍

- ***Your Design***
- image
- ***Figma Design***
- image

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @montsecruz9 :wave: 
 
Well done on implementing the changes. 👍 
Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: Descriptive pull request.
:heavy_check_mark: Correct GitHub flow.
:heavy_check_mark: Commit messages are meaningful.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 10 above)
Comment (montsecruz9) [reply index.html:246]: Thanks @Tufahel !! But i had that fixed already before. See: 
image

**Pull request merged**